react-accordion/Accordion/src/TopicCard.tsx

```diff
@@ -0,0 +1,24 @@
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+type Props = {
+  topics: Topic[];
+  isActive: number;
+  onShow: (index: number) => void;
+};
+
+export function TopicCard({ topics, isActive, onShow }: Props) {
```

Comment:
My expectation is that `TopicCard` displays just _one_ topic. Then all the list logic you have here goes up in Accordion instead.